bin/archive-demo/src/main.rs

```diff
@@ -0,0 +1,55 @@
+use desub::{Chain, Decoder};
+use subxt::{
+	backend::{
+		legacy::{
+			rpc_methods::{Bytes, NumberOrHex},
+			LegacyRpcMethods,
+		},
+		rpc::{rpc_params, RpcClient},
+	},
+	config::PolkadotConfig,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+	pretty_env_logger::init();
+
+	// Connect to a node with an RPC client:
+	let rpc_client = RpcClient::from_url("wss://rpc.polkadot.io").await?;
+	let methods = LegacyRpcMethods::<PolkadotConfig>::new(rpc_client.clone());
+
+	let mut bn = 1;
```

Comment:
I think `block_number` is easier to understand here,

My initial take was `big number` but clearly wrong :P

Reply:
Agreed :)